workers/email.js

```diff
@@ -1,7 +1,7 @@
-var fs = require('fs'),
-    nconf = require('nconf');
-    mandrill = require('mandrill-api'),
-    _ = require('lodash');
+import fs from 'fs';
+import nconf from 'nconf';
+import mandrill from 'mandrill-api';
+import _ from 'lodash';
```

Comment:
Looks like we only use `find` and `findIndex` here, can we import those individually?